apps/antalmanac/src/components/RightPane/CoursePane/SchoolDeptCard.tsx

```diff
@@ -47,38 +48,28 @@ class SchoolDeptCard extends PureComponent<SchoolDeptCardProps> {
 
     render() {
         const html = { __html: this.props.comment };
-        const ExpandIcon = this.state.commentsOpen ? ExpandLess : ExpandMore;
         return (
             <Grid item xs={12}>
-                <Paper className={this.props.classes[this.props.type]} elevation={1} square>
-                    <Typography
-                        noWrap
-                        variant={this.props.type === 'school' ? 'h6' : 'subtitle1'}
-                        className={this.props.classes.text}
-                    >
-                        {this.props.name}
-                    </Typography>
-                    <>
-                        <ExpandIcon
-                            onClick={() =>
-                                this.setState({
-                                    commentsOpen: !this.state.commentsOpen,
-                                })
-                            }
-                            className={this.props.classes.icon}
-                        />
-
-                        <Collapse in={this.state.commentsOpen} className={this.props.classes.collapse}>
-                            <Typography variant="body2">
-                                {this.props.comment === '' ? 'No comments found' : 'Comments:'}
+                <Paper elevation={1} square>
+                    <Accordion>
+                        <AccordionSummary expandIcon={<ExpandMore />}>
+                            <Typography 
+                                variant={this.props.type === 'school' ? 'h6' : 'subtitle1'}
+                            >
+                                {this.props.name}
                             </Typography>
-                            <div dangerouslySetInnerHTML={html} className={this.props.classes.comments} />
-                        </Collapse>
-                    </>
+                        </AccordionSummary>
+                        <AccordionDetails>
+                                <Typography variant="body2">
+                                    {this.props.comment === '' ? 'No comments found' : 'Comments:'}
+                                    <Box dangerouslySetInnerHTML={html} className={this.props.classes.comments} />
+                                </Typography>
```

Comment:
```suggestion
                                <Typography variant="body2">
                                <span>
                                    {this.props.comment === '' ? 'No comments found' : 'Comments:'}</span>
                                    <Box dangerouslySetInnerHTML={html} className={this.props.classes.comments}  component="p" />
                                </Typography>
```
Box is a `div` by default I believe; divs shouldn't be nested inside of typographic elements. You can set the `component` prop for Box to make it assume something like a regular paragraph. Also, if there are multiple elements, I like to explicitly wrap sections with some tag so that way it's easier to distinguish them.